### ISSUE TITLE
Alarm clock is confusing

### ISSUE THREAD

**Issue by spaetz** [segler-alex/RadioDroid#64]
Hi, thanks for radiodroid!

When I select the tab "alarm clock" (Wecker), I see an empty screen with no buttons and no help text. I thought something is broken until I selected a radio stream in the main view  and added an alarm through its context menu.

Woiuld it be possible to a) either display a gray help text on the alarm page when no alarm is set (something like "no alarm has been set yet. Select a screen and chose 'wakeup")' or b) add a + button on thjis page which would let you select a radioo station which you want to add as alarm.

**Comment by Neudrino** [comment]
I agree with @spaetz . I found it hard to set an alarm. I actually had to search this bug tracker to find out how to do it. I was now lucky enough that @spaetz not only pointed out the problem, but also how to actually set an alarm. (And that only 2 days ago. Lucky me! :smile: Thanks!)

A help text within the alarm menu is probably the quickest help for confused users in short term, however for user experience a + button would certainly be nice, such that it behaves like all the other alarm apps.
